frontend/public/components/chargeback.tsx

```diff
@@ -151,17 +150,19 @@ const DataTable = ({rows, orderBy, sortBy, applySort, keys, maxValues, totals}:D
   const className = `col-md-${size}`;
   return <div className="co-m-table-grid co-m-table-grid--bordered" style={{marginTop: 20, marginLeft: -15, marginRight: -15}}>
     <ListHeader>
-      {_.map(keys, k => <ColHead
-        className={classNames(className, {'text-right': REDUCER_COLS.indexOf(k) < 0})}
-        key={k}
-        sortField={k}
-        sortFunc={k}
-        currentSortOrder={orderBy}
-        currentSortField={sortBy}
-        currentSortFunc={sortBy}
-        applySort={applySort}>
-        {k.replace(/_/g, ' ')}
-      </ColHead>)}
+      {_.compact(_.map(keys, k => { // never show ColHeader for data we don't have
+        return _.get(rows, [0, k]) !== undefined ? <ColHead
```

Comment:
Do we only know at runtime whether a column will have data? Is it possible to simply add more columns to the black list?

Reply:
The aggregation of the data will remove data/columns from the dataset.  aggregation only deals with finite numbers and removes anything else (strings, dates, etc..)

Reply:
Now it is just a safety check to ensure we have the data we think we should have.

Reply:
I'm confused why we need this sanity check... If it fails, won't the columns headers no longer match the row data? Is there ever a time we expect this to be undefined?

Reply:
Removed